notebooks/MAST/astroqueryMAST/beginner_program_id_search/beginner_program_id_search.ipynb

```diff
@@ -0,0 +1,402 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Option 2: Store Token\n",
+    "Observations.login(store_token=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using 'store_token' will allow us to automatically log in, without needing to re-enter the token, for as long as the token remains valid. Note that tokens expire after 10 days of inactivity, or 60 days after creation, whichever comes first. Once it expires, you should use `reenter_token=True` to overwrite the old token with the new one.\n",
+    "\n",
+    "The third option is to store the token as the bash environment variable `$MAST_API_TOKEN`. This method varies from system to system; for more details, you can check out [this guide](https://www3.ntu.edu.sg/home/ehchua/programming/howto/Environment_Variables.html) (links to a non-STScI site).\n",
+    "\n",
+    "Let's take a minute to verify that our login was successful:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session_info = Observations.session_info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see all of your information above. If not, verify that your token and MyST account are active.\n",
+    "\n",
+    "And of course, if the need arises, we can logout:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Observations.logout()\n",
+    "session_info = Observations.session_info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Searching for Data by ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "We can use a program ID to query the MAST archives for data. In the example below, we'll use 2733 as the ID. This is the program that produced the [stunning images](https://webbtelescope.org/contents/media/images/2022/033/01G709QXZPFH83NZFAFP66WVCZ) of the Southern Ring Nebula!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Let's get a list of all observations associated with this proposal\n",
+    "obs_list = Observations.query_criteria(proposal_id=2733)\n",
+    "\n",
+    "# We can chooose the columns we want to display in our table\n",
+    "disp_col = ['dataproduct_type','calib_level','obs_id','target_name','filters','proposal_pi']\n",
```

Comment:
if you add "obs_collection" to the list of columns, then it becomes really easy to identify the Public Outreach image (obs_collection = "OPO") from the mission data (obs_collection = "JWST"), and then describing the difference based on a lack of PI unnecessary and can instead be based no filled-in values instead of lack-of-values.